spec/requests/preferences/preference_spec.rb

```diff
@@ -81,4 +81,68 @@
       end
     end
   end
+
+  describe 'GET edit' do
+    context 'when logged in' do
```

Comment:
What do you think about adding a `context` for not logged scenario? 😉

Reply:
That sounds great! I will add it to a new commit and let you know.